src/Lookup/Lookup.tsx

```diff
@@ -489,7 +493,7 @@ export default class Lookup extends React.PureComponent<LookupProps, State> {
 			if (!results.options) {
 				results.options = [];
 			}
-			const hasMore = results.options.length > 0;
+			const hasMore = results.options.length > 0 && results.options.length === this.props.pageSize;
```

Comment:
This seems to be totally erroneous. When you scrolled the results and retrieved 2 pages this will compute `hasMore=false` but there can be more pages

Reply:
No it will not? If the pageSize askes for 30 and got 25 results, why should it expect more pages?

Reply:
![image](https://user-images.githubusercontent.com/2778221/80476213-5b442800-894a-11ea-816d-25e22f61cddc.png) With the fix
Total requests here is 11

Without the fix
![image](https://user-images.githubusercontent.com/2778221/80476270-7151e880-894a-11ea-9795-65327d78486b.png)
Total request here are 12. The last one returned no results.

Reply:
Sorry, you are right. I mixed up results.options with all current options in menu